### PR TITLE
docs: Add che-code repository

### PR DESCRIPTION
**Pull request by benoitf**
### What does this PR do?
Add che-code repository

### What issues does this PR fix or reference?
N/A

### How to test this PR?
Just look at the Contributing.MD file

### PR Checklist

[As the author of this Pull Request I made sure that:](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#pull-request-template-and-its-checklist)

- [x] [The Eclipse Contributor Agreement is valid](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-eclipse-contributor-agreement-is-valid)
- [x] [Code produced is complete](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-produced-is-complete)
- [x] [Code builds without errors](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#code-builds-without-errors)
- [x] [Tests are covering the bugfix or new feature ](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#tests-are-covering-the-bugfix-or-new-feature)
- [x] [The repository devfile is up to date and works](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#the-repository-devfile-is-up-to-date-and-works)
- [x] [Sections `What issues does this PR fix or reference` and `How to test this PR` completed](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#sections-what-issues-does-this-pr-fix-or-reference-and-how-to-test-this-pr-completed)
- [x] [Relevant user documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [x] [Relevant contributing documentation updated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#relevant-contributing-documentation-updated)
- [x] [CI/CD changes implemented, documented and communicated](https://github.com/eclipse/che/blob/master/CONTRIBUTING.md#cicd-changes-implemented-documented-and-communicated)

### Reviewers

Reviewers, please comment how you tested the PR when approving it.

Change-Id: Ieb4b7434be58335d1c438238b2364bdd4bf2841d
Signed-off-by: Florent Benoit <fbenoit@redhat.com>

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -22,7 +22,16 @@ Repository | Component | Description | Devfile | Documentation
 ---| [tests](https://github.com/eclipse/che/tree/master/tests) | source code of our integration tests. | | |
 [che-theia](https://github.com/eclipse-che/che-theia) | | Theia IDE integrated in Che. | [devfile](https://github.com/eclipse-che/che-theia/blob/main/devfiles/che-theia-all.devfile.yaml) | [doc](https://github.com/eclipse-che/che-theia/blob/main/CONTRIBUTING.md)
 ---| [generator](https://github.com/eclipse-che/che-theia/tree/main/generator) | `che:theia init` CLI to prepare and build che-theia | | |
-[chectl](https://github.com/che-incubator/chectl) | | The CLI to install Che, create and start workspaces and devfiles | [devfile](https://github.com/che-incubator/chectl/blob/master/devfile.yaml) | [doc](https://github.com/che-incubator/chectl/blob/master/CONTRIBUTING.md)
+[chectl](https://github.com/che-incubator/chectl
+https://github.com/che-incubator/che-code) | | The CLI to install Che, create and start workspaces and devfiles
```
Comment (azatsarynnyy) [inline CONTRIBUTING.md]: `The CLI to install Che ...` - is something wrong here? :slightly_smiling_face:

Comment (benoitf) [reply CONTRIBUTING.md]: yes the script failed due to a typo in the yaml file

Comment (benoitf) [reply CONTRIBUTING.md]: should be good now 👍 
thanks for the report

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -23,6 +23,7 @@ Repository | Component | Description | Devfile | Documentation
 [che-theia](https://github.com/eclipse-che/che-theia) | | Theia IDE integrated in Che. | [devfile](https://github.com/eclipse-che/che-theia/blob/main/devfiles/che-theia-all.devfile.yaml) | [doc](https://github.com/eclipse-che/che-theia/blob/main/CONTRIBUTING.md)
 ---| [generator](https://github.com/eclipse-che/che-theia/tree/main/generator) | `che:theia init` CLI to prepare and build che-theia | | |
 [chectl](https://github.com/che-incubator/chectl) | | The CLI to install Che, create and start workspaces and devfiles | [devfile](https://github.com/che-incubator/chectl/blob/master/devfile.yaml) | [doc](https://github.com/che-incubator/chectl/blob/master/CONTRIBUTING.md)
+[che-code](https://github.com/che-incubator/che-code) | | Fork of code-oss to work with Eclipse Che | [devfile](https://github.com/che-incubator/che-code/blob/master/devfile.yaml) | [doc](https://github.com/che-incubator/che-code/blob/master/README.md)
```
Comment (azatsarynnyy) [inline CONTRIBUTING.md]: > `code-oss`

Officially it's called `Code - OSS`. I think it matters here.

File: CONTRIBUTING.md (hunk 1 above)
Comment (azatsarynnyy) [reply CONTRIBUTING.md]: no problem )

File: CONTRIBUTING.md (hunk 2 above)
Comment (benoitf) [reply CONTRIBUTING.md]: updated again

**Review by azatsarynnyy — APPROVED** [review]

**Pull request merged**